Sources/Fog/View/FogView+TxOutFetcher.swift

```diff
@@ -65,14 +65,16 @@ extension FogView {
             }
             fogViewService.query(requestWrapper: requestWrapper) {
                 {
-                    guard let highestProcessedBlockCount = try? $0.get().highestProcessedBlockCount else {
+                    guard let highestProcessedBlockCount = try?
+                                $0.get().highestProcessedBlockCount
+                    else {
```

Comment:
The wrapped let assignment seems overly indented... not sure what the best way to handle this is, but I did see some examples that looked interesting...

One was to have the let start on its own line, and to wrap the else like you did as well.

Wdyt of this for this case?

```suggestion
                    guard
                        let highestProcessedBlockCount = try? $0.get().highestProcessedBlockCount
                    else {
                        return
                    }
```